egs/arctic/vc/local/text

```diff
@@ -0,0 +1 @@
+../../../ljspeech/tts1/local/text
```

Comment:
Now `local/text` is integrated with `espnet_tts_frontend`.
We do not need to use it (Maybe it does not exist).